### ISSUE TITLE
DLSS Not available 

### ISSUE THREAD

**Issue by SethizFTW** [sultim-t/prboom-plus-rt#13]
I am able to run the game but I cannot turn on DLSS.
![image](https://user-images.githubusercontent.com/30537722/161386936-05d4fa5b-5c55-4163-8922-318259a8a4e3.png)
I am running an RTX 3050TI. I do find this funny because Doom Eternal I can't use RT but I can use DLSS and in this game it is the opposite.

**Comment by arrowgent** [comment]
read the release
https://github.com/sultim-t/prboom-plus-rt/releases/tag/rt-1.0

```
DLSS:
download [nvngx_dlss.dll](https://github.com/NVIDIA/DLSS/blob/main/lib/Windows_x86_64/rel/nvngx_dlss.dll), and place it to the target folder```

**Comment by SethizFTW** [comment]
> read the release https://github.com/sultim-t/prboom-plus-rt/releases/tag/rt-1.0
> 
> ```
> DLSS:
> download nvngx_dlss.dll, and place it to the target folder
> ```

Oh I am silly, thanks.

**Comment by Jaeonn** [comment]
what is the "target folder" is it the folder prboom-plus.exe is in?

**Comment by sultim-t** [comment]
Yes

**Comment by clyder** [comment]
I also cannot enable Nvidia DLSS. I have a 2070RTX. I see it in a supported card list.
I placed both the music folder and nvngx_dlss.dll in the same folder prboom-plus.exe is.
Graphics drivers are updated, I tried running it as administrator and even tried manualy changing cfg file and rt_dlss property to something other than 0 (tried 1 and 2) and nvidia dlss is always grayed out.
I can use AMD fsr though.
![settings](https://user-images.githubusercontent.com/1040296/161435437-2ee0d834-5bd5-4414-aa14-84a056e87be4.png)
![files](https://user-images.githubusercontent.com/1040296/161435440-d2b5dd08-b68d-46d4-9da3-9e5840dd5759.png)
